spec/features/proposals_spec.rb

```diff
@@ -203,6 +203,89 @@
         expect(page).not_to have_link("No comments", href: "#comments")
       end
     end
+
+    scenario "Shows clickable markers for related proposals by tag on proposal map", :js do
+      Setting["feature.map"] = true
+      create(:tag, :category, name: "culture")
+      create(:tag, :category, name: "sports")
+      proposal = create(:proposal, tag_list: "culture, sports", map_location: create(:map_location))
```

Comment:
Since we add a `latitude` to `related_proposal`, how about adding `latitude: 51.48` to `proposal`? I know it's the default value, but I didn't know it the first time I read this line and I won't remember it the next time either :smile:.